src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -107,6 +107,75 @@ namespace {
         return usernameText;
     }
 
+    void appendTwitchEmoteOccurrences(const QString &emote,
+                                      std::vector<TwitchEmoteOccurrence> &vec,
+                                      const std::vector<int> &correctPositions,
+                                      const QString &originalMessage,
+                                      int messageOffset)
+    {
+        auto *app = getIApp();
+        if (!emote.contains(':'))
+        {
+            return;
+        }
+
+        auto parameters = emote.split(':');
+
+        if (parameters.length() < 2)
+        {
+            return;
+        }
+
+        auto id = EmoteId{parameters.at(0)};
+
+        auto occurences = parameters.at(1).split(',');
```

Comment:
Can the other `occurence` spelling mistakes be corrected or is it already too late?